### DISCUSSION TITLE
Docker issue on Synology NAS

### DISCUSSION THREAD

**Discussion by marcbaier** [jasonacox/Powerwall-Dashboard#169] [Q&A] [answered]
I was able to connect to my NAS via SSH and installed the powerwall-dashboard on a separate share called powerwall. I am now running the ./setup.sh command from /volume1/powerwall/powerrwall-dashboard and am getting the error: ERROR: docker is not available or not runnning.
This script requires docker, please install and try again.

But I have already installed Docker via the regular package center on the NAS. 

Can anyone help me with what I might be missing ? Docker is running on the NAS. 
<img width="1137" alt="Screen Shot 2023-02-02 at 11 49 39 AM" src="https://user-images.githubusercontent.com/97667985/216304832-438336de-383a-4c00-b5db-ff767e7e5294.png">

**Comment by jasonacox** [comment, marked as answer]
Hi @marcbaier  - there have been others who have done this on a Synology NAS, check out issue https://github.com/jasonacox/Powerwall-Dashboard/issues/22#issuecomment-1110099800

The steps that seem key are these (but check out the rest of the issue thread to see if that helps):

```bash
# Log in via SSH to Synology DSM. Create a usergroup docker:
sudo synogroup --add docker

# Change ownership of docker to this group:
sudo chown root:docker /var/run/docker.sock

# Make the user a member of the newly created group:
sudo synogroup --member docker <user>
```
